### PR TITLE
PA-2352

### PR DESCRIPTION
**Pull request by devinleighsmith**
Add last updated date corresponding to tenancy field. Automatically updated by FE when tenancy field updated. Also add to review page. Includes required backend changes.

### CODE REVIEW

**Review by Fosol — APPROVED** [review]

File: frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.scss [hunk 1]
```diff
@@ -51,6 +51,9 @@
   .harmful {
     justify-content: center;
   }
+  .form-control {
+    background-image: unset;
```
Comment (agalbrai) [inline frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.scss:55]: what does this do?

File: frontend/src/utils/YupSchema.ts [hunk 2]
```diff
@@ -135,6 +135,7 @@ export const BuildingSchema = Yup.object().shape({
     .min(0, 'Floor Count must be a valid number')
     .transform(emptyStringToNull),
   buildingTenancy: Yup.string().max(100, 'Tenancy must be less then 100 characters'),
+  buildingTenancyUpdatedOn: Yup.string().required('Required'),
```
Comment (agalbrai) [inline frontend/src/utils/YupSchema.ts:138]: updatedOn date is required but the tenancy field is not required?

File: frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.scss (hunk 1 above)
Comment (devinleighsmith) [reply frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.scss:55]: removes the green check icon on the review page, as it frequently disrupts the text content due to the short widths.

File: frontend/src/utils/YupSchema.ts (hunk 2 above)
Comment (devinleighsmith) [reply frontend/src/utils/YupSchema.ts:138]: right, good catch, need this to be required only when the tenancy is specified.

**Review by agalbrai — APPROVED** [review]

**Pull request merged**